### DISCUSSION TITLE
unit 3.3 equation question

### DISCUSSION THREAD

**Discussion by jihobak** [Lightning-AI/dl-fundamentals#45] [Lecture videos and quizzes] [answered]
when updating weight, isn't it right to do minus with gradient?

or did you mean alpha is minus value?

![image](https://github.com/Lightning-AI/dl-fundamentals/assets/10293523/2a822aed-484f-43fe-9849-cbae932aa393)

**Comment by rasbt** [comment, marked as answer]
Yes, thanks, it should be 

$$w + \Delta w$$

where 

$$\Delta w = - \alpha \nabla L_w$$

**Reply by rasbt** [reply]
Added it to the Errata sheet [here](https://github.com/Lightning-AI/dl-fundamentals/blob/main/errata/README.md) and will update the slide in the video in the upcoming weeks. Thanks again!
